### ISSUE TITLE
New buttons for Organic

### ISSUE THREAD

**Issue by DeRobyJ** [LMMS/lmms#968]
Hi, Just tried 1.0.92 with fixed Organic. It's awesome and I've got some little ideas to make it even better!

The randomize button creates a whole new organ ready to be played, by setting defaults Harm/Pad/Wide and actually randomizing only Wave/Vol.

Three things:
- Randomize Line: If you click on "Pan", all the pans are randomized, and the same for all the other lines of the matrix. This will actually make it possible to obtain the old creepy sounds by clicking "Wide".
- Randomize Oscillator: if you click on the number of an oscillator, all its values are randomized.
- Spacial-ize button: a new button just over the Randomize one that will put the "Pan" like so:
  1: -100
  2: -66
  3: -33
  4: 0
  5: 0
  6: 33
  7: 66
  8: 100
  Maybe this should also set all volumes to default... or maybe not, since it would change the sound further.

I'm so exited about Organic, it's so cool now!

----

Edit: fixing some grammar 10 years later

**Comment by DeRobyJ** [comment]
> Is this issue actual? I see Randomize button even in 1.2.2 version ...

Hi! I unfortunately did not take the challenge at the time, and my career doesn't align well with C++ and Qt. I still look forward to the day I will come back and contribute to the software I love, but it hasn't been the past 10 years and it won't be now either

So I guess it's up to anyone else who feels like implementing the changes 18yo me tought would have been cool 😁

**Comment by zonkmachine** [comment]
> Is this issue actual?
> I see Randomize button even in 1.2.2 version ...

I don't think so. Did you try it? Please read the issue again and then fire up LMMS and try Organic some more.

**Comment by firewall1110** [comment]
Thank You for quick answer!
Now I understand - it is actual.
